src/quo2/components/wallet/wallet_activity/component_spec.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.wallet.wallet-activity.component-spec
+  (:require [quo2.components.wallet.wallet-activity.view :as wallet-activity]
+            [test-helpers.component :as h]))
+
+(def props
```

Comment:
nit, is this def necessary? - seems easier t to use an empty map and avoid the assoc on line 16